src/corespotlight.cs

```diff
@@ -979,6 +989,41 @@ interface CSSearchableItemAttributeSet : NSCopying, NSSecureCoding {
 		[Export ("valueForCustomKey:")]
 		[return: NullAllowed]
 		INSSecureCoding ValueForCustomKey (CSCustomAttributeKey key);
+
+		// CSSearchableItemAttributeSet_CSGeneral
+
+		[iOS (11,0), NoTV, Mac (10, 11)]
+		[NullAllowed, Export ("userCreated", ArgumentSemantic.Strong)]
+		[Protected] // We would like to use [BindAs (typeof (bool?))]
```

Comment:
make those `[Internal]` so we can switch to `[BindAs]` without having to keep binary compatibility with those symbols